pipeline/tasks/protoc_tasks.py

```diff
@@ -344,3 +346,24 @@ def execute(self, language, api_name, output_dir, src_proto_path,
         if repo_dir:
             arg_list += ['-r', repo_dir]
         self.run_packman(*arg_list)
+
+
+class GoExtractImportBaseTask(task_base.TaskBase):
```

Comment:
It is only used by the Grpc and Core generation pipelines - it depends whether we are classifying the tasks based their use case, or based on what resources they require.